deps/CMakeLists.txt

```diff
@@ -1,14 +1,8 @@
-if(OS_WINDOWS)
-  add_subdirectory(ipc-util)
+if(OS_WINDOWS AND NOT MINGW)
```

Comment:
We do not support compiling obs-studio via MINGW, so any breakage incurred here is desired.

Reply:
Looks like it's a remnant of the past for `w32-pthread` through my rebase.
Because while I rebased `NOT MINGW` was there to skip `w32-pthread`.

Reply:
Should be good now.